web/src/components/layout/layout.tsx

```diff
@@ -26,7 +26,7 @@ import LocalizationSelect from './localization-select';
 import Logo from './logo';
 import Nav from './nav';
 import UserMenu from './user-menu';
-import * as cx from 'classnames';
+import cx from 'classnames';
```

Comment:
Typescript couldn't find the types when we imported it with `* as`

Reply:
I wonder why it was fine prior to adding a linter.

Reply:
I think just some combination of the config files meant Typescript wasn't working correctly which is weird.